### PR TITLE
Tip changes

### LINKED ISSUE #254: Tip changes
Remove default message and if success, don't show inputs

### PR DESCRIPTION
**Pull request by mike-gusev-ae**
close #254

### CODE REVIEW

**Review by DeeJayElly — APPROVED** [review]

File: react/features/aeternity/components/TipButton.js [hunk 1]
```diff
@@ -287,7 +288,8 @@ class TipButton extends Component<Props, State> {
         const { t } = this.props;
         const amount = aeternity.util.aeToAtoms(this.state.value);
         const url = `${URLS.SUPER}/user-profile/${this.props.account}`;
-        const { message } = this.state;
+        const DEFAULT_MESSAGE = `Appreciation from conference : ${APP.conference.roomName} on ${window.location.host}.`;
```
Comment (venimus) [inline react/features/aeternity/components/TipButton.js]: we don't need that text anymore

Comment (mike-gusev-ae) [reply react/features/aeternity/components/TipButton.js]: but if i'm not wrong, you and Yuri said that we need comment while we dont get new version of contract (or something like that)

Comment (venimus) [reply react/features/aeternity/components/TipButton.js]: yes, but the field is required, so ppl will have to type something anyway

Comment (mike-gusev-ae) [reply react/features/aeternity/components/TipButton.js]: okay, got it

**Review by venimus — APPROVED** [review]

**Pull request merged**